client/src/templates/Challenges/classic/classic.css

```diff
@@ -91,6 +91,14 @@
   overflow-y: auto;
 }
 
+[data-uses-multifile-editor='true'][data-has-editable-boundaries='true']
+  #mobile-layout
+  .tab-content {
+  height: calc(
+    100vh - var(--header-height, 0px) - var(--flash-message-height, 0px) - 33px
+  ); /* What does 33px represent? */
```

Comment:
Is this addressing a specific issue?

Reply:
Yes, it fixes the double scroll bar issue in the steps while viewing in the mobile layout. Something changed though recently (I think it has to do with the bread crumb links being moved in the DOM) and so the `33px` value was no longer correct and I updated it. You will notice that when viewing the steps in the mobile layout there is only one vertical scroll bar now (on the editor).

I also adjusted the height for the non-steps in mobile layout but it only works correctly in Firefox. In Chrome-based browsers there is something else going on that is causing a very big vertical scroll bar that doesn't need to be there. Do you want me to try and fix that in this PR?

Reply:
Thanks for looking into that. 

I see that the tool panel no longer sticks to the bottom of the display and would require extra scrolling.

I could not replicate the double scroll bar issue on the new responsive web design challenges. 

The behavior seems to be the same when I disable the newly added styles completely. 

If the fix is more involved,  I think we should open an issue for the double scroll and the hidden tool panel and tackle it in another PR.

Reply:
> If the fix is more involved, I think we should open an issue for the double scroll and the hidden tool panel and tackle it in another PR.

Yes, I think it might be best to fix all of the scrolling issues first and then we can remove the tool panel from the mobile layout. I will open a new issue to document all of the current scrolling issues.

Reply:
This scrolling issue has been fixed in another PR and this discussion is no longer applicable.